compoments/report-repair/summary.js

```diff
@@ -0,0 +1,51 @@
+import React, {useEffect, useState} from 'react';
+import SummaryList from '../summaryList';
+import Button from '../button';
+const Summary = ({values, changeLinkUrlValues}) => {
+  let [whatIsRepairProblemLink, setWhatIsRepairProblemLink] = useState(null)
+  let [describeRepairProblemLink, setDescribeRepairProblemLink] = useState(null)
+  useEffect(()=>{
+    setWhatIsRepairProblemLink(changeLinkUrlValues[values.repairLocation])
+    setDescribeRepairProblemLink(changeLinkUrlValues[values.repairProblem])
+    console.log('ADDRESS',values.address)
+  })
+  const personalDetailsSummary = [
+    {pageName:'Repair address', value: values.address+ ',' + values.postcode, link: 'postcode'},
+    {pageName:'Appointment contact number', value: values.contactPersonNumber, link: 'contact-person'},
+  ]
+  const repairDetailsSummary = [
+    { pageName:'Where is the problem?', value: values.repairLocation, link:'repair-location'},
+    { pageName:'What is the problem?', value: values.repairProblem, link: changeLinkUrlValues[values.repairLocation]},
+    { pageName:'What best describes the problem?', values: values.repairProblemBestDescription, link: changeLinkUrlValues[values.repairProblem]},
```

Comment:
The "What best describes the problem?" answer is blank even though it is mandatory a selection would've been made.

![image](https://user-images.githubusercontent.com/1875867/145792239-ac33e090-7898-463b-93e0-6e6ee60ee03b.png)